tests/test_venv.py

```diff
@@ -303,20 +291,15 @@ def test_env_variables_added_to_needs_reinstall(tmpdir, mocksession, newconfig,
     assert env["TEMP_NOPASS_VAR"] == "456"
 
 
-def test_test_hashseed_is_in_output(newmocksession):
-    original_make_hashseed = tox.config.make_hashseed
-    tox.config.make_hashseed = lambda: '123456789'
-    try:
-        mocksession = newmocksession([], '''
-            [testenv]
-        ''')
-    finally:
-        tox.config.make_hashseed = original_make_hashseed
+def test_test_hashseed_is_in_output(newmocksession, monkeypatch):
+    seed = '123456789'
+    monkeypatch.setattr('tox.config.make_hashseed', lambda: seed)
+    mocksession = newmocksession([], "")
```

Comment:
``mock_session``?

Reply:
glueing words together like this is like a @hpk42 trademark, so I am just going with conventions here :)

Reply:
I beg to differ, given @hpk42 is not actively maintaining this anymore I would prefer us to have Pythonic names now, and yes that involves changing some conventions here and there.